src/app/components/CodeDisplay/index.tsx

```diff
@@ -71,13 +71,22 @@ type RawDataDisplayProps = {
   extraTopPadding?: boolean
 }
 
-export const RawDataDisplay: FC<RawDataDisplayProps> = ({ data, label, extraTopPadding }) => {
-  const code = data === undefined ? undefined : base64ToHex(data)
-  if (!code) {
-    return null
-  }
+export const RawDataDisplay: FC<RawDataDisplayProps & { decodeBase64?: boolean }> = ({
+  data,
+  label,
+  decodeBase64 = false,
+}) => {
```

Comment:
Instead of changing this: would be better to create a new component around CodeDisplay